bolts/data/datasets/vision/celeba.py

```diff
@@ -67,11 +67,11 @@ class CelebASplit(Enum):
 class CelebA(PBVisionDataset):
     """CelebA dataset."""
 
-    """The data is downloaded to `download_dir` / `_BASE_FOLDER`."""
+    # The data is downloaded to `download_dir` / `_BASE_FOLDER`.
```

Comment:
Why do you prefer strings to comments?

Reply:
I have no preference, but PEP8 does. `"""` is for docstrings, and `#` is for comments

Reply:
Aren't these comments and not docstrings?

Reply:
https://www.python.org/dev/peps/pep-0008/#comments

Reply:
Okay?

Reply:
Oh sorry I read the change the wrong way round...